pages/customer-type.js

```diff
@@ -20,9 +20,12 @@ const CustomerType = (props) => (
       validatorErrors={props.validatorErrors}
       onHandleErrorClick={OnHandleErrorClick}
     />
+    <Fieldset>
+    <Fieldset.Legend>
     <Heading as="h1" size="LARGE">
```

Comment:
Instead of having a Heading within the Legend, you should be able to add the appropriate attributes to the Legend element:
```
<Fieldset.Legend
            size="LARGE"
            isPageHeading
            style={{ marginBottom: "30px" }}
          >
           {props.t("Who will this establishment supply food to?")}
          </Fieldset.Legend>
```